modules/plugin/svg/src/main/java/org/geotools/renderer/style/RenderableSVG.java

```diff
@@ -0,0 +1,105 @@
+/* (c) 2017 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geotools.renderer.style;
+
+import java.awt.Graphics2D;
+import java.awt.geom.AffineTransform;
+import java.awt.geom.Rectangle2D;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import org.apache.batik.bridge.BridgeContext;
+import org.apache.batik.bridge.DocumentLoader;
+import org.apache.batik.bridge.GVTBuilder;
+import org.apache.batik.bridge.UserAgent;
+import org.apache.batik.bridge.UserAgentAdapter;
+import org.apache.batik.gvt.GraphicsNode;
+import org.w3c.dom.Document;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+class RenderableSVG {
+    Rectangle2D bounds;
+
+    GraphicsNode node;
+
+    public RenderableSVG(Document doc) {
+        this.node = getGraphicNode(doc);
+        this.bounds = getSvgDocBounds(doc);
+        if (bounds == null) bounds = node.getBounds();
```

Comment:
Missing braces, I'm not sure if the original GT code guidelines still apply since we move to google format.

Reply:
Actually Java code ask for them but apparently doesn't enforces them:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used